cmd/gemini/jobs.go

```diff
@@ -147,8 +200,9 @@ func job(ctx context.Context, f testJob, actors uint64, schema *gemini.Schema, s
 		UseLWT:          schemaConfig.UseLWT,
 	}
 
-	for j, table := range schema.Tables {
+	for j := range schema.Tables {
 		gen := generators[j]
+		table := schema.Tables[j]
```

Comment:
This one is actually bugfix, not just `linter complain addressing`.
So, looks like it must be in a separate PR.

Reply:
moved to separate commit